UnitTest.Commands/source/empty.json

```diff
@@ -0,0 +1 @@
+2020-04-06 14:16:56 4 ERROR - AppInsLog - Command not supported for DLL calls
```

Comment:
This file doesn't seem like it should be checked in

Reply:
The unit tests need a source file that will result in no scan matches.  For the TagTest command two different files are needed as it will reject testing for delta's against the same one hence both an empty and blank named source set of files are checked in.  These two files are copied however.  A new commit has removed unneeded versions of these files which already exists in the desired locations under source\unzipped\simple.

Reply:
You could also use `Path.GetTempFileName()` to get a blank file. [Path.GetTempFileName](https://docs.microsoft.com/en-us/dotnet/api/system.io.path.gettempfilename?view=netframework-4.8)

If indeed these files are intended to have no matches, having any content in them seems non optimal.